src/SecurityRulesEnabled.ruleset

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0"?>
+<RuleSet Name="Security Rules Enabled with default severity" Description="All Security Rules are enabled with default severity. Security Rules with IsEnabledByDefault = false are force enabled with default severity. Rules from a different category are disabled." ToolsVersion="15.0">
+   <!-- Security Rules -->
```

Comment:
This file is taken directly from the `SecurityRulesEnabled.ruleset` that comes with the `Microsoft.CodeAnalysis.FxCopAnalyzers` version `3.3.0-beta2.final` nuget.